app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx

```diff
@@ -343,16 +356,20 @@ function PrepareToRun({
 
   return (
     <>
+      {/* Empty box to detect scrolling */}
+      <Flex ref={scrollRef} />
       {/* Protocol Setup Header */}
       <Flex
+        boxShadow={isScrolled ? BORDERS.shadowBig : undefined}
         flexDirection={DIRECTION_COLUMN}
         gridGap={SPACING.spacing24}
-        paddingBottom={SPACING.spacing40}
-        paddingTop={SPACING.spacing32}
+        padding={`${SPACING.spacing32} ${SPACING.spacing40} ${SPACING.spacing40} ${SPACING.spacing40}`}
```

Comment:
top, left&right, bottom
```suggestion
        padding={`${SPACING.spacing32} ${SPACING.spacing40} ${SPACING.spacing40}`}
```